### PR TITLE
Main Page Frontend

### PR DESCRIPTION
**Pull request by mawada-sweis**
I provide these features:
- You can upload an image by Choose File button.
- You can drag and drop image instead of uploading it using the button.
- The image will display after uploading it.
- The predict plate number button will displayed when the image shown.
- For now, the predict plate number button will display a message instead of do the actual function.

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html>
+	<head>
+		<title>Car Plate Recognition</title>
+		<link rel="stylesheet" type="text/css" href="../static/css/style.css">
+	</head>
+	<body>
+		<div class="container">
+      <h1>Car Plate Recognition</h1>
+      <div class="form-group">
+        <label for="upload">Select an car image to upload:</label>
```
Comment (zubaidasader) [inline templates/index.html:11]: or u  can say: "upload a car image"

**Review by zubaidasader — APPROVED** [review]

File: templates/index.html (hunk 1 above)
Comment (mawada-sweis) [reply templates/index.html:11]: I use 'select' term to avoid the misleading

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Car Plate Recognition</title>
+		<link rel="stylesheet" type="text/css" href="../static/css/style.css">
+	</head>
+	<body>
+		<div class="container">
+      <h1>Car Plate Recognition</h1>
+      <div class="form-group">
```
Comment (Rama-Has) [inline templates/index.html:10]: "VEhicle Plate Number Recognition" is the right title.

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <label for="upload">Select an car image to upload:</label>
+        <input type="file" accept="image/*" id="upload" onchange="loadFile(event)">
```
Comment (Rama-Has) [inline templates/index.html:12]: Can u please put the input inside a form with a submit button?

**Review by Rama-Has — CHANGES_REQUESTED** [review]

**Review by Rama-Has — APPROVED** [review]

**Pull request merged**